src/core_plugins/metrics/public/kbn_vis_types/editor_controller.js

```diff
@@ -1,6 +1,6 @@
 import modules from 'ui/modules';
 import '../services/executor';
-import 'plugins/timelion/directives/refresh_hack';
+import '../../../timelion/public/directives/refresh_hack';
```

Comment:
i am not sure this is good idea .... if something changes in the refresh hack for timelion this will break ... we basically have a coupling between timelion and tsvb which should not be there. however the only other way i see is code duplication which i don't like either. up to you to decide, maybe ask for another review, maybe @Bargs ?